### ISSUE TITLE
Google Home support

### ISSUE THREAD

**Issue by NovaGL** [coldfire84/node-red-alexa-home-skill-v3-web#36]
Can you give me more info on how Google Home support works.

Does the payload say if it's for Google Home or Alexa?

I'm sure it's in the pipeline but the wiki needs to be updated to advise which services support Google Home and which support Alexa.

**Comment by NovaGL** [comment]
Thanks. I think I meant response. Is that the same too?

I guess the best way is to try and report back once I'm in the beta.

**Comment by coldfire84** [comment]
> Thanks. I think I meant response. Is that the same too?

Yes, the response process is the same. Internally the API handles the commands (and responses) from each of the digital assistants separately. Node output is consistent though, irrespective of command source.

I'll extend the beta shortly.

**Comment by coldfire84** [comment]
@NovaGL as discussed, skill extended. Note that for your HVAC unit to work via Google Home it will need the [FanSpeed](https://developers.google.com/actions/smarthome/traits/fanspeed) trait, which I'll be adding alongside the Alexa [RangeController](https://developer.amazon.com/docs/device-apis/alexa-rangecontroller.html) capability next week.

**Comment by NovaGL** [comment]
Thanks, how do I discover devices on Google Home? I can't seem to see anything. Can you make sure the skill is enabled for English-AU.

**Comment by coldfire84** [comment]
1. Open the Google Home App
2. Hit Account icon (bottom right)
3. Under Account go to Settings
4. Under Settings go to Assistant
5. Under Assistant go to Home Control
6. Under Home Control open top-left menu and select "Manage account"

It will show linked/ unlinked services (plus others). Hit the "[test] Node RED Smart Home Control" service and press "Unlink account".

**Comment by NovaGL** [comment]
Thanks. Looks like I can't link it then. Every time I click it goes to try and link it again.

**Comment by coldfire84** [comment]
@NovaGL I've "published" a new draft version to the testing group. Let me know if you have any more success in linking accounts. Also extended the Slack invite for the skill development.
